docs/api-docs/apps/guide/apps-03-developing.md

```diff
@@ -4,13 +4,107 @@
 
 ### On This Page

```

Comment:
The section titles here and throughout the article should be in [sentence case](https://intranet.bigcommerce.com/display/SST/Headings+and+Titles).